Test/src/main/resources/mods.groovy

```diff
@@ -61,16 +75,21 @@ ModsDotGroovy.make {
             intermediate_mappings = "net.fabricmc:intermediary"
         }
 
-        // Quilt entrypoints
+        // Quabric entrypoints
         entrypoints {
-            init = adapted {
-                value = 'test.no.No'
+            onQuilt {
+                init = adapted {
+                    value = 'test.no.No'
+                }
+            }
+            onFabric {
+                main = 'mymod.Hi'
             }
-            client_init = 'test.no.NoClient'
+            client 'test.no.NoClient'
```

Comment:
MDG uses setter syntax everywhere, accepting a List or Closure if multiple values can be specified. Please change this back to a setter.